app/tests/test_routes.py

```diff
@@ -13,3 +13,68 @@ def test_base_route():
 
     assert response.status_code == 200
     assert response.get_data() == b'try the predict route it is great!'
+
+def test_post_applicants_route_invalid_format():
+    app = Flask(__name__)
+    configure_routes(app)
+    client = app.test_client()
+    url = '/applicants'
+
+    response = client.post(url, json=100)
+
+    assert response.status_code == 404 # Not found: age, absences, and health attributes
+
+
+def test_get_applicants_route_invalid_request():
+    app = Flask(__name__)
+    configure_routes(app)
+    client = app.test_client()
+    url = '/applicants'
+
+    response = client.get(url)
+
+    assert response.status_code == 404 # Not found: age, absences, and health attributes
+
+
+def test_post_applicants_route_invalid_input():
+    app = Flask(__name__)
+    configure_routes(app)
+    client = app.test_client()
+    url = '/applicants'
+
+    response = client.post(url, json={
+        'age': 16, 'absences': 3
+    })
+
+    assert response.status_code == 404 # Not found: health attribute
+
+
+def test_predict_route_invalid_request():
+    app = Flask(__name__)
+    configure_routes(app)
+    client = app.test_client()
+    url = '/predict'
+
+    response = client.get(url)
+
+    assert response.status_code == 500 # Internal server error: nothing was posted
+    
+
+def test_applicants_and_predict_route():
```

Comment:
Hey Grace, this all looks good but can you separate the tests for /predict and /applicants here? Because if this test function fails we want to know exactly which endpoint is failing. Combining the 2 might take some time for us to figure out exactly what line and might be unclear.